alma/common/clear_history.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -ex
+
+# Sync yum and rpmdb after installing rpm's outside yum
+yum history sync
```

Comment:
Could this be generalized to a common clear_history script for multiple distributions?

Reply:
Good idea to have a common file for clear history. I will handle this in the next PR (this would need changes to pipelines as well)